src/components/macro/macro.component.ts

```diff
@@ -1,31 +1,36 @@
-import { Component, OnDestroy, OnInit, QueryList, ViewChildren } from '@angular/core';
+import { Component, QueryList, ViewChildren } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 
-import { Subscription } from 'rxjs/Subscription';
+import { Store } from '@ngrx/store';
+import 'rxjs/add/operator/let';
+import 'rxjs/add/operator/switchMap';
 
```

Comment:
I imported it manually and now it's working. When I did it automatically I was getting error that this operator doesn't exist, even though it's part of the documentation.